osbuild.spec

```diff
@@ -37,6 +37,7 @@ Requires:       tar
 Requires:       util-linux
 Requires:       python3-%{pypi_name} = %{version}-%{release}
 Requires:       (%{name}-selinux if selinux-policy-%{selinuxtype})
+Requires:       python3-librepo
```

Comment:
Is `librepo` available on all RPM-based systems?

Reply:
My quick research showed that it's available on Fedora and its enterprise kids, Ubuntu and Arch. Good enough from my perspective. :)